kedro/contrib/io/pyspark/spark_jdbc.py

```diff
@@ -144,6 +144,12 @@ def __init__(
 
         # Update properties in load_args and save_args with credentials.
         if credentials is not None:
+
+            # Check credentials for bad inputs
+            for k, v in credentials.items():
+                if v is None:
+                    raise DataSetError(f"Credential property `{k}` cannot be empty. Please provide a value")
```

Comment:
```suggestion
                    raise DataSetError(f"Credential property `{k}` cannot be empty. Please provide a value.")
```
Trivial suggestion to end with a period, consistent with surrounding error messages.